great_expectations/init_notebooks/create_expectations.ipynb

```diff
@@ -17,26 +17,8 @@
    "cell_type": "markdown",
```

Comment:
Removed most of the text (since it would not be read anyway) and replaced it with links to documentation.

This notebook template is not completely done yet, since the documentation URLs do not exist and it points to a wrong tutorial video at the top.